redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/data/twoImages.json

```diff
@@ -0,0 +1 @@
+[{ "Id": "file1" }, { "Id": "file2" }]
```

Comment:
Shouldn't the values be `ContentVersion` Ids'?

Reply:
No, they're mocks. in this case it doesn't matter.

Reply:
Seriously. They're mocks. It's best practice to use mocks, and there's no better way to convey to our users that we're using mocks, than by using obviously false data as the mocks. I get the argument for consistency, but this is wrong.